### PR TITLE
copy update /openstack/managed

### LINKED ISSUE #10805: [COPY UPDATE] managed - "Managed OpenStack on Ubuntu"
Please update managed - "Managed OpenStack on Ubuntu" from the [copy doc](https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit?usp=drivesdk)

### PR DESCRIPTION
**Pull request by goulinkh**
## Done

- Update the page /openstack/managed as requested in the [copy update](https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/managed or https://ubuntu-com-10809.demos.haus/openstack/managed
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please mark the discussion as resolved in the [copy doc](https://docs.google.com/document/d/1g38PPPhEP0Z66eM_gezfUyUwAr5e9vGfTHJDWV8gZlE/edit#) once the QA is done

## Issue / Card

Fixes #10805

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

**Comment by goulinkh** [comment]
@tytus-kurek Can you please provide an asset for the missing asset in the copy?

**Comment by tytus-kurek** [comment]
@goulinkh I responded in the copy doc

**Comment by goulinkh** [comment]
Waiting for illustration

File: templates/openstack/managed.html [hunk 1]
```diff
@@ -1,725 +1,610 @@
...
+    <h2>Why managed private cloud?</h2>
     <div class="col-8">
-      <h2>Managed private cloud, a perfect fit for multi-cloud environments</h2>
-      <p>We recommend a multi-cloud operations strategy for every company running applications at scale. The combination of public clouds and efficient, automated private infrastructure, ideally with outsourced and standardised operations under the hood, offers the best mix of latency and price across both fixed and variable workloads.
-        <a href="/engage/multi-cloud-guide">We understand why no one type of cloud architecture or infrastructure is ideal for everyone</a>
+      <p>While public clouds provide immediate access to the infrastructure-as-a-service (IaaS) offering, they are not economical in the long term and at scale. At the same time, implementing a private cloud comes with a cost.</p>
+      <p>
+        What if:
+        <ul>
+          <li>Your public cloud costs are growing with a trajectory you cannot afford?</li>
```
Comment (minkyngkm) [inline templates/openstack/managed.html:109]:
```suggestion
          <li class="p-list__item is-ticked">Your public cloud costs are growing with a trajectory you cannot afford?</li>
```

File: templates/openstack/managed.html [hunk 2]
```diff
@@ -1,725 +1,610 @@
...
+      }}
+    </div>
   </div>
+</section>
 
+<section class="p-strip">
   <div class="u-fixed-width">
-    <p><a href="/openstack/contact-us?product=openstack-managed-cloud" class="p-button--positive js-invoke-modal">Let's discuss your requirements</a></p>
+    <h2>Learn more about Managed OpenStack</h2>
+    <div class="p-divider row">
```
Comment (minkyngkm) [inline templates/openstack/managed.html:538]:
```suggestion
    <div class="p-divider row u-no-padding--left u-no-padding--right">
```

File: templates/openstack/managed.html [hunk 3]
```diff
@@ -1,725 +1,610 @@
...
-    <div class="col-4 p-card">
-      <h4>Public cloud</h4>
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">Elasticity for highly variable load</li>
-        <li class="p-list__item is-ticked">Many regions for low latency</li>
-        <li class="p-list__item is-ticked">Choice of providers</li>
-        <li class="p-list__item is-ticked">OPEX as needed</li>
-      </ul>
+    <div class="col-4">
+      {{
```
Comment (minkyngkm) [inline templates/openstack/managed.html:120]: The image doesn't look clear. Could it be updated with the better resolution? Also, should it be aligned vertically centered?

**Review by minkyngkm — CHANGES_REQUESTED** [review]
Looks good @goulinkh ! just left some comments with suggestions! :)

File: templates/openstack/managed.html [hunk 4]
```diff
@@ -1,725 +1,610 @@
...
   <div class="u-fixed-width">
-    <ul class="p-matrix is-trisected">
-      <li class="p-matrix__item">
-        <div class="p-matrix__content">
-          <h3 class="p-matrix__title p-heading--four">Predictable CAPEX and OPEX</h3>
-          <p class="p-matrix__desc">Dedicated hardware is the most cost-effective for long-term workloads. Managed OpenStack costs are predictable per server per month. Transfer control at any time.</p>
-        </div>
-        </li>
+    <h2>What is Managed OpenStack?</h2>
+    <p>Managed OpenStack is a fully-managed service for <a href="/openstack/managed">Charmed OpenSack</a> provided by Canonical. We build the cloud and we manage it for you. From 24x7 monitoring, through daily maintenance tasks to regular upgrades. We focus on your cloud, you focus on your business.</p>
```
Comment (minkyngkm) [inline templates/openstack/managed.html:179]:
```suggestion
    <p>Managed OpenStack is a fully-managed service for <a href="/openstack/managed">Charmed OpenStack</a> provided by Canonical. We build the cloud and we manage it for you. From 24x7 monitoring, through daily maintenance tasks to regular upgrades. We focus on your cloud, you focus on your business.</p>
```

File: templates/openstack/managed.html [hunk 5]
```diff
@@ -1,725 +1,610 @@
...
+        <li class="p-list__item is-ticked">Optional Managed Kubernetes service on top</li>
+        <li class="p-list__item is-ticked">EU and US regulatory compliance options</li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <h2>Modernise your infrastructure with Managed OpenStack</h2>
```
Comment (minkyngkm) [inline templates/openstack/managed.html:513]: maybe we need to wrap the text with `<div class="col-8">` and the image with `<div class="col-4">`?

File: templates/openstack/managed.html [hunk 6]
```diff
@@ -1,725 +1,610 @@
...
-                alt="",
-                height="72",
-                width="195",
+                url="https://assets.ubuntu.com/v1/87106ba5-Tele2.png",
+                alt="Tele2",
+                height="140",
+                width="355",
                 hi_def=True,
-                loading="lazy",
+                loading="auto|lazy",
```
[the remark this answers is not in the document]
Comment (minkyngkm) [reply templates/openstack/managed.html:32]: Should we add this attrs?
`attrs={"class": "logo-section__logo"}`

File: templates/openstack/managed.html (hunk 1 above)
Comment (sowasred2012) [reply templates/openstack/managed.html:109]: I think in this case tick icons aren't appropriate - they indicate something positive, but this is a list problems/negative things.

File: templates/openstack/managed.html [hunk 7]
```diff
@@ -1,725 +1,610 @@
...
   <div class="row">
+    <h2>Why managed private cloud?</h2>
     <div class="col-8">
-      <h2>Managed private cloud, a perfect fit for multi-cloud environments</h2>
-      <p>We recommend a multi-cloud operations strategy for every company running applications at scale. The combination of public clouds and efficient, automated private infrastructure, ideally with outsourced and standardised operations under the hood, offers the best mix of latency and price across both fixed and variable workloads.
-        <a href="/engage/multi-cloud-guide">We understand why no one type of cloud architecture or infrastructure is ideal for everyone</a>
+      <p>While public clouds provide immediate access to the infrastructure-as-a-service (IaaS) offering, they are not economical in the long term and at scale. At the same time, implementing a private cloud comes with a cost.</p>
+      <p>
+        What if:
+        <ul>
```
[the remark this answers is not in the document]
Comment (sowasred2012) [reply templates/openstack/managed.html:108]: See comment below re: tick icons, there isn't an icon that would suit this, so the standard `ul` is OK in this instance.

File: templates/openstack/managed.html [hunk 8]
```diff
@@ -1,725 +1,610 @@
...
+        <li class="p-list__item is-ticked">Proactive cloud management</li>
+        <li class="p-list__item is-ticked">Incident and problem resolution</li>
+        <li class="p-list__item is-ticked">Software updates and OpenStack upgrades</li>
+        <li class="p-list__item is-ticked">SLA and regular independent audits</li>
+        <li class="p-list__item is-ticked">Most cost-effective approach up to 300 nodes</li>
+        <li class="p-list__item is-ticked">Optional Managed Kubernetes service on top</li>
+        <li class="p-list__item is-ticked">EU and US regulatory compliance options</li>
+      </ul>
+    </div>
+  </div>
```
Comment (sowasred2012) [inline templates/openstack/managed.html:508]: Immediate descendants of `.row` elements should always be a `.col-X` element, in this case I think you could achieve the layout with:
```suggestion
  <div class="u-fixed-width">
    <h2>Managed OpenStack pricing</h2>
  </div>
  <div class="row">
    <div class="col-6 p-card">
      <h3 class="p-heading--4">$15 per server per day</h3>
      <hr class="u-sv1" />
      <p>What's included:</p>
      <ul class="p-list">
        <li class="p-list__item is-ticked">Logging, monitoring and alerting</li>
        <li class="p-list__item is-ticked">24x7 operations with fully staffed NOC</li>
        <li class="p-list__item is-ticked">Proactive cloud management</li>
        <li class="p-list__item is-ticked">Incident and problem resolution</li>
        <li class="p-list__item is-ticked">Software updates and OpenStack upgrades</li>
        <li class="p-list__item is-ticked">SLA and regular independent audits</li>
        <li class="p-list__item is-ticked">Most cost-effective approach up to 300 nodes</li>
        <li class="p-list__item is-ticked">Optional Managed Kubernetes service on top</li>
        <li class="p-list__item is-ticked">EU and US regulatory compliance options</li>
      </ul>
    </div>
  </div>
```

File: templates/openstack/managed.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (sowasred2012) [reply templates/openstack/managed.html:108]: @minkyngkm See comment below re: tick icons, there isn't an icon that would suit this, so the standard ul is OK in this instance.

File: templates/openstack/managed.html (hunk 1 above)
Comment (minkyngkm) [reply templates/openstack/managed.html:109]: > I think in this case tick icons aren't appropriate - they indicate something positive, but this is a list problems/negative things.

@sowasred2012 yep, right. That makes sense!

File: templates/openstack/managed.html (hunk 6 above)
[the remark this answers is not in the document]
Comment (goulinkh) [reply templates/openstack/managed.html:32]: Good point!

File: templates/openstack/managed.html (hunk 4 above)
Comment (goulinkh) [reply templates/openstack/managed.html:179]: Well seen!

File: templates/openstack/managed.html (hunk 5 above)
Comment (goulinkh) [reply templates/openstack/managed.html:513]: The image will not be aligned vertically with images from other sections

File: templates/openstack/managed.html (hunk 3 above)
Comment (goulinkh) [reply templates/openstack/managed.html:120]: I added the new illustration

File: templates/openstack/managed.html [hunk 9]
```diff
@@ -1,725 +1,618 @@
...
+      url="https://assets.ubuntu.com/v1/24a03775-Contact+us.svg",
+      alt="",
+      height="171",
+      width="240",
+      hi_def=True,
+      loading="lazy",
+      ) | safe
+      }}
+    </div>
+  </div>
```
Comment (sowasred2012) [inline templates/openstack/managed.html:540]: @minkyngkm is right, but you'd need a couple extra classes on the `col-4` to vertically center the image. See the section titled "OpenStack ensures maximum multi-cloud cost optimisation" on https://ubuntu.com/openstack/what-is-openstack for another example.

```suggestion
  <div class="row">
    <div class="col-8">
      <h2>Modernise your infrastructure with Managed OpenStack</h2>
      <p>Canonical's Managed OpenStack brings your infrastructure costs down, while not putting an overhead on your
        internal operations team. Talk to one of our cloud experts to get cost estimates for Managed OpenStack and choose
        the package tailored to your needs.</p>
      <p><a href="/openstack/contact-us" class="p-button--positive">Talk to an expert</a></p>
    </div>
    <div class="col-4 u-align--center u-vertically-center u-hide--small">
      {{
      image(
      url="https://assets.ubuntu.com/v1/24a03775-Contact+us.svg",
      alt="",
      height="171",
      width="240",
      hi_def=True,
      loading="lazy",
      ) | safe
      }}
    </div>
  </div>
```

File: templates/openstack/managed.html [hunk 10]
```diff
@@ -1,725 +1,618 @@
...
-    <div class="col-3 col-start-large-9">
-      {% include "openstack/shared/_openstack-pie-chart.html" %}
-    </div>
-  </div>
-</section>
-<section class="p-strip--light is-deep is-bordered">
+<section class="p-strip">
   <div class="row">
+    <h2>Why managed private cloud?</h2>
     <div class="col-8">
```
Comment (sowasred2012) [inline templates/openstack/managed.html:110]: From [Vanilla docs](https://vanillaframework.io/docs/patterns/grid):
> 
    - The page structure must be laid out using rows (.row)
    - All content must be within columns (.col-*)
    - Only columns should be direct children of a row

`u-fixed-width` is a shorthand way of expressing `.row>.col-12`, so we could get around this like this:
```suggestion
  <div class="u-fixed-width">
    <h2>Why managed private cloud?</h2>
  </div>
  <div class="row">
    <div class="col-8">
```

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
Looking good, just a couple of minor changes

File: templates/openstack/managed.html (hunk 10 above)
Comment (goulinkh) [reply templates/openstack/managed.html:110]: Sorry didn't see this one

File: templates/openstack/managed.html [hunk 11]
```diff
@@ -1,725 +1,619 @@
...
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Modernise your infrastructure with Managed OpenStack</h2>
   </div>
+  <div class="row">
+    <div class="col-8">
```
Comment (sowasred2012) [inline templates/openstack/managed.html:523]: Sorry, I think the heading should be within the 8 col, it looks a little odd with the title hanging over the image like that:

```suggestion
  <div class="row">
    <div class="col-8">
      <h2>Modernise your infrastructure with Managed OpenStack</h2>
```

**Review by sowasred2012 — APPROVED** [review]
One final change, and then it's good to go!

File: templates/openstack/managed.html (hunk 11 above)
Comment (goulinkh) [reply templates/openstack/managed.html]: Thanks Scott!

**Review by minkyngkm — APPROVED** [review]

**Comment by sowasred2012** [comment]
@tytus-kurek would you mind giving this a review?

**Comment by tytus-kurek** [comment]
@sowasred2012 LGTM. Thank you very much to all of you for your hard work and making it happen!

**Pull request merged**